.travis.yml

```diff
@@ -4,56 +4,30 @@ sudo: false
 python:
   - "2.7"
   - "3.4"
+  - "3.5"
 
-# command to install dependencies, e.g. pip install -r requirements.txt --use-mirrors
+cache:
```

Comment:
it would be nice to test that testdb.sqlite is generated correctly. But it is really a pain. Is there any way to say: start from scratch only if ncbiquery.py file has changes? Probably asking too much :)